src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,9 +3,39 @@
 public class Comment{
     private final String name;
     private final String comment;
+    private final long id;
 
-    public Comment(String name, String comment){
+    public Comment(String name, String comment, long id){
         this.name = name;
         this.comment = comment;
+        this.id = id;
     }
+
+    public String getDisplayName(){
+        return name; 
+    }
+
+    public String getComment(){
+        return comment; 
+    }
+
+    public long getID(){
+        return id; 
+    }
+
+    @Override
+    public boolean equals(Object o) {   
+        if (o == this) { 
```

Comment:
I don't know if I made the comment about equals on this PR or another but:

o==this block is not necessary, please remove it.

typically we do 

Comment that = (Comment) o;
return this.getComment().equals(that.getComment())
           && this.getDisplayName().equals(that.getDisplayName());

(use the this/that pattern).

You'll notice that you have to use the equals methods to test equality for Strings.

Also why does this not test the ID?

Reply:
doesn't check ID cause doesn't need to be the same entity in datastore - mainly for testing purposes